### PR TITLE
Adding Implementation of few sorting algorithms

### PR DESCRIPTION
**Pull request by Ahel2000**
# Description

Created a separate folder called SortingAlgorithms under src.
Added 5 sorting algorithms in Java- Merge sort, Quick sort, Selection sort, Insertion sort and Shell sort- each under separate folder.
Will add the explanations by this week.

 #5 

## Type of change

Please delete options that are not relevant.

- [x] Added Algorithm
- [x] Folder Structure

# Added to SUMMARY.md (delete section if not needed)?

Have you added the page to SUMMARY.md if this is a new page and not a fix?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by arnabsen1729** [comment]
@Ahel2000 please go through the README.md and CONTRIBUTING.md.  

For a file to be rendered on the website it must be in markdown format, not as a native file. Instead, you have to make separate md files for each topic, add some brief description about the sorting algo (that helps in searching related keywords), also it is advised to build the src locally using `mdbook build -o` and check for errors. 

If you still have some questions ask in the channel.

File: src/SortingAlgorithms/SORTING.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (arnabsen1729) [inline src/SortingAlgorithms/SORTING.md:1]: Do not add empty markdown files, Make a list of href links to all the sorting algos you are specifying.

File: src/SortingAlgorithms/SelectionSort/Selection.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+
+/**
```
Comment (arnabsen1729) [inline src/SortingAlgorithms/SelectionSort/Selection.java:2]: remove unnecessary comments.

**Comment by Ahel2000** [comment]
Okay I will make the changes.

**Pull request closed without merging**